src/components/Home/OptionsForm.tsx

```diff
@@ -1,12 +1,60 @@
+import { TextInputSize } from '@commonComponents/TextInput/TextInput.d';
 import { OptionsFormContainer } from './Home.styles';
+import ColorInput from '@commonComponents/ColorInput/ColorInput';
+import TextInput from '@commonComponents/TextInput/TextInput';
+import FileInput from '@commonComponents/FileInput/FileInput';
+import { useState } from 'react';
+
+interface QRForm {
+  size: string;
+  color: string;
+  background: string;
+  file: File | undefined;
+}
 
 const OptionsForm = () => {
+  const [form, setForm] = useState<QRForm>({
+    size: '0px',
+    color: '',
+    background: '',
```

Comment:
We should set the default colors instead of an empty string. The string should never be empty.

Reply:
Lets set the default size to something like 200px.